datanode/gateway/graphql/schema.graphql

```diff
@@ -1184,6 +1184,16 @@ type Query {
     pagination: Pagination
   ): TeamRefereeHistoryConnection
 
+  "Time weighted notional position is a metric used to determine if a reward should be paid to a party"
+  timeWeightedNotionalPosition(
+    "Asset the position is denominated in"
+    assetId: ID!
+    "Party holding the position"
+    partyId: ID!
```

Comment:
Its possibly scope creep for this PR.. but I'd like to get the queries that take partyId to be consistently able to accept multiple partyIds. We have a bit of a mixture in the graphql/rest APIs with regard to this so its fine if this isn't done now

Reply:
@mattrussell36 Could you create an issue for that, ideally listing as many queries as possible where this would need to be added, because as it stands I would agree that this is scope-creep :)